.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-  - "8.9.4"
+  - "10.15.0"
```

Comment:
Can this be just "10", please?

Reply:
It's better to specify the exact version in the CI build to not break it

Reply:
OK, so we would probably enforce the same on the local build using engine?
So let's use "10.9.0" as main dependency:
https://github.com/angular/angular/blob/master/.nvmrc#L1

Reply:
Done